engine/src/main/resources/assets/shaders/chunk_frag.glsl

```diff
@@ -290,8 +287,8 @@ void main() {
     gl_FragData[0].rgb = color.rgb;
     // Encode occlusion value into the alpha channel
     gl_FragData[0].a = occlusionValue;
-    // Encode daylight value into the normal alpha channel
-    gl_FragData[1].a = daylightValue;
+    // Encode shininess value into the normal alpha channel
```

Comment:
Store shininess here because the normal texture gets sent to the geometry shader.